### ISSUE TITLE
Ignore specific files

### ISSUE THREAD

**Issue by djibe** [validator/validator#1049]
Hi,
nowhere in the doc I can find how to ignore a specific file:

I get many errors with my Google validation file:

googlebdb6f8bf41a883bb.html":1.1-1.52: error: Element "head" is missing a required instance of child element "title".
googlebdb6f8bf41a883bb.html":1.1-1.52: error: Non-space characters found without seeing a doctype first. Expected "<!DOCTYPE html>".
googlebdb6f8bf41a883bb.html":1.1-1.52: info warning: Consider adding a "lang" attribute to the "html" start tag to declare the language of this document.

Is it a new feature or not specified in doc ?
Thanks a lot

**Comment by sideshowbarker** [comment]
No, there is not mechanism in the checker code itself to allow you to specify particular files you want to ignore. And doing that it kind of beyond the scope of what the checker is about. There are all kinds of other existing external ways to ignore certain files before you pass them on to the checker.

Anyway, given that this isn’t really in scope for the checker, I’m going ahead and closing it.
